src/app/debugger.js

```diff
@@ -0,0 +1,21 @@
+var $ = require('jquery');
+
+function Debugger (executionContext, _id) {
+  this.el = document.querySelector(_id);
+  this.debugger = new window.remix.Debugger(executionContext.web3());
```

Comment:
I think this is the easier solution.
here is how remix is loaded initially
https://github.com/ethereum/remix/blob/master/src/index.js

using something like require('ethereum-remix') // with ethereum-remix refering to the (remix) app.js was giving error while compiling browser solidity (i think because app.js is already a browserified file).

using something like require('ethereum-remix') // with ethereum-remix refering directly to ./ethdebugger.js
is not possible because in that case string templates are not yo-yoified (and string template are not supported by a few browsers)

@axic any thoughts about that? is there a better way?